elytron/WFCORE-4496-principal-transformer-in-aggregate-realm.adoc

```diff
@@ -0,0 +1,60 @@
+= WFCORE-4496 Adding principal-transformer in aggregate-realm
+:author:            Ashley Abdel-Sayed
+:email:             aabdelsa@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Elytron currently has an aggregate security realm which is a combination of two realms; an authentication and an
+authorization realm. There is currently no way to transform the principal in between the two realms.
```

Comment:
It would be good to mention that this new attribute will be a reference to a ```principal-transformer``` that has been defined in the ```mappers``` configuration in the Elytron subsystem.